frontend/javascripts/admin/voxelytics/task_list_view.tsx

```diff
@@ -452,6 +458,13 @@ export default function TaskListView({
     );
   };
 
+  const totalRuntime = report.run.tasks
+    .filter((t) => "endTime" in t && "beginTime" in t)
+    .reduce(
+      (sum, t) => sum.add(moment.duration(moment(t.endTime).diff(moment(t.beginTime)))),
+      moment.duration(0),
+    );
```

Comment:
I think it would be cool to also show the time for tasks that are still running.

```suggestion
  const totalRuntime = report.run.tasks
    .reduce(
      (sum, t) => {
        if (t.state === VoxelyticsRunState.RUNNING) {
          return sum.add(moment.duration(moment().diff(moment(t.beginTime))));
        } else if (t.beginTime != null && t.endTime != null) {
          return sum.add(moment.duration(moment(t.endTime).diff(moment(t.beginTime))));
        } else {
          return sum;
        }
      },
      moment.duration(0),
    );
```